packages/ai/src/ai/common/database/db_instance_base.py

```
@@ -41,54 +41,208 @@
+            raise ValueError('"question" is required and must be a non-empty string')
+
+        question = question.strip()
+        limit = args.get('limit', 250)
+        result = self._buildSQLQuery(question, limit=limit)
+
+        is_valid = result.get('isValid', '').lower() == 'true'
+        sql_query = result.get('query', '')
+
+        if is_valid and sql_query and is_sql_safe(sql_query):
+            return {'sql': sql_query, 'valid': True}
+        elif is_valid and sql_query:
```

Ruff (0.15.7)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)